File: src/C#/Creational Patterns/Factory/CreditCards/Creators/MoneyBackCreditCardCreator.cs
using CreditCards.Products;
using System;
using System.Collections.Generic;
using System.Text;

namespace CreditCards.Creators
{
    public class MoneyBackCreditCardCreator : ICreditCardCreator
    {
        public CreditCard CreateCreditCard(string model, decimal limit, decimal annualCharge)
            => new MoneyBack(model, limit, annualCharge);
    }
}

File: src/PHP/Creational Patterns/Factory/CreditCards/Creators/MoneyBackCreditCardCreator.php
<?php

namespace CreditCards\Creators;

use CreditCards\Products\MoneyBack;

class MoneyBackCreditCardCreator implements ICreditCardCreator
{
    public function createCreditCard($model, $limit, $annualCharge): MoneyBack
    {
        return new MoneyBack($model, $limit, $annualCharge);
    }
}